input.css

```diff
@@ -0,0 +1,19 @@
+.input-container {
+ background-color: lightgray;
+width:  31.25rem;
+height  : 3.125rem;
+border-radius: 0.625rem;
+border: 0.125rem solid black;
+padding: 0.625rem 1.25rem 0.625rem 1.25rem;
```

Comment:
for padding you can use the another shorthand like `padding: 0.625rem 1.25rem`
can you try using units in `px` in cases where the value  is < `1rem`

Reply:
i dont want to use px in my code as of now. done! Thanks for the review